src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -367,7 +367,7 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
         input = symbolResult.io == IoInput // input = true, if db contains "INPUT"
       ),
       wdlType,
-      symbolResult.wdlValue.map(wdlType.fromWdlString)
+      symbolResult.wdlValue.map {value => wdlType.fromWdlString(value.toRawString)}
```

Comment:
out of the various maps in here this is the other one i'll continue to grumble about since this is the other spot where you're transitioning from one of the two common styles to that hybrid. 

My own personal preference is towards `foo.map(likeThis)` but @mcovarr pointed out that one of the few things the official style guide is opinionated about is that it is `foo map likeThis` so I've been trying to keep it that way.

I'm not certain that the doing it `foo.map {likeThis}` is exactly the same as their example (which is `foo.map (likeThis)`) but note that in that block on the style guide they explicitly warn against that pattern as being "shunned outright": http://docs.scala-lang.org/style/method-invocation.html#higherorder_functions